share/metainfo/org.kitone.subtitleeditor.appdata.xml

```diff
@@ -13,16 +14,18 @@
             to synchronize subtitles to voices.
         </p>
     </description>
-    <screenshots>
-        <screenshot type="default">https://kitone.github.io/subtitleeditor/docs/screenshoot.png</screenshot>
-    </screenshots>
-    <url type="homepage">https://kitone.github.io/subtitleeditor/</url>
-		<updatecontact>kitone at free dot fr</updatecontact>
-		<keywords>
-			<keyword>subtitles</keyword>
-			<keyword>subtitle</keyword>
-			<keyword>captions</keyword>
-			<keyword>video</keyword>
-		</keywords>
-</application>
+	<screenshots>
+		<screenshot type="default">
+			<image>http://home.gna.org/subtitleeditor/screenshots/se-0.34-01.png</image>
```

Comment:
You change the url with the old one
https://kitone.github.io/subtitleeditor/docs/screenshoot.png